src/appkit.cs

```diff
@@ -3829,6 +3886,72 @@ partial interface NSColor : NSCoding, NSCopying, NSSecureCoding, NSPasteboardRea
 		[Static]
 		[Export ("scrubberTexturedBackgroundColor", ArgumentSemantic.Strong)]
 		NSColor ScrubberTexturedBackgroundColor { get; }
+
+		[Mac (10,13)]
+		[Static]
+		[Export ("colorNamed:bundle:")]
+		[return: NullAllowed]
+		NSColor FromName (string name, [NullAllowed] NSBundle bundle);
+
+		[Mac (10,13)]
+		[Static]
+		[Export ("colorNamed:")]
+		[return: NullAllowed]
+		NSColor FromName (string name);
+
+		[Mac (10, 13)]
+		[Export ("type")]
+		NSColorType Type { get; }
+
+		[Mac (10,13)]
+		[Export ("colorUsingType:")]
+		[return: NullAllowed]
+		NSColor GetColor (NSColorType type);
```

Comment:
Because that was what @spouliot suggest after the first pass. I think I like FromType better though.